package.json

```
@@ -26,15 +26,13 @@
   ],
   "scripts": {
     "build": "npm run clean && npm run build-ts && npm run fix-react-shadow",
-    "build-all": "npm run build-app && npm run build-stories && mv build dist/stories",
+    "build-all": "npm run build-app && npm run build-storybook && mv build dist/stories",
```

You can test that this script works locally

```suggestion
    "build-all": "npm run build-app && npm run build-storybook && mv storybook-static dist/stories",
```

---

It works thanks